<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgInterface</class>
 <widget class="QDialog" name="DlgInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit interface configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GoupInterface">
     <property name="title">
      <string>Network interface</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="LabelName">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="buddy">
         <cstring>EditName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="EditName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LabelHWaddress">
        <property name="text">
         <string>MAC address:</string>
        </property>
        <property name="buddy">
         <cstring>EditHWaddress</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="EditHWaddress">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="LabelIP">
        <property name="text">
         <string>Current IP:</string>
        </property>
        <property name="buddy">
         <cstring>EditIP</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="EditIP">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Disconnected</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="LabelCustomName">
        <property name="text">
         <string>Custom name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="EditCustomName">
        <property name="placeholderText">
         <string>Custom name you can set to improve readability</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupPredefinedIP">
     <property name="title">
      <string>Predefined IP</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="TablePredefinedIP">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Custom name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IP</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Network mask</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Gateway</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="VLayoutMove">
        <item>
         <widget class="QPushButton" name="ButtoNewIP">
          <property name="text">
           <string>New IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonEditIP">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>EditIP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonDeleteIP">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete IP</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="VSpacerTable">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonUp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonDown">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <widget class="QPushButton" name="ButtonForgetInterface">
       <property name="text">
        <string>Forget interface configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
